docs/IE_PLUGIN_DG/plugin_transformation_pipeline/low_precision_transformations/pipeline/step4_cleanup.md

```diff
@@ -2,6 +2,10 @@
 
 @sphinxdirective
 
+.. meta::
+   :description: Check the list of transformations used to clean up the 
+                 resulting model to avoid not handled dequantization operations.
```

Comment:
```suggestion
.. meta::
   :description: Check the list of transformations used to clean up the 
                 resulting model to avoid unhandled dequantization operations.
```